android/envoy/build.gradle

```diff
@@ -65,9 +64,6 @@ dependencies {
     implementation 'androidx.core:core-ktx:1.3.2'
     implementation 'androidx.preference:preference-ktx:1.1.1'
     implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'
-    //embed project(path: ':cronet', configuration: 'release')
-    releaseImplementation project(path: ':cronet', configuration: 'release')
-    debugImplementation project(path: ':cronet', configuration: 'debug')
```

Comment:
Should we keep the local `debugImplementation` for debug builds? IIRC only the release build is on Maven